src/job-manager/plugins/dws-jobtap.c

```diff
@@ -0,0 +1,122 @@
+/************************************************************\
+ * Copyright 2021 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+/* dws-test.c - keep jobs in depend state if they contain a dws attribute,
+ *  send an RPC for validation, and wait for a response to release
+ */
+
+#if HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include <stdio.h>
+#include <jansson.h>
+
+#include <flux/core.h>
+#include <flux/jobtap.h>
+
+#define VALIDATE_DEP_NAME "dws-test"
+
+struct validate_arg_t {
+    flux_plugin_t *p;
+    flux_jobid_t id;
+};
+
+static void validate_cb (flux_future_t *f, void *arg)
+{
+    struct validate_arg_t *args = arg;
+    int success = false;
+    char *errstr = NULL;
+    flux_t *h = flux_future_get_flux(f);
+    char jobid_buf[64];
+
+    flux_job_id_encode (args->id, "f58", jobid_buf, 64);
+    if (flux_rpc_get_unpack (f,
+        "{s:b, s?s}",
+        "success", &success,
+        "errstr", &errstr) < 0)
```

Comment:
We've unofficially adopted a style of aligning function arguments on newlines with the first argument, e.g.:
```suggestion
    if (flux_rpc_get_unpack (f,
                             "{s:b, s?s}",
                             "success", &success,
                             "errstr", &errstr) < 0)
```

Optional, since this is a separate repo